app/uk/gov/hmrc/incorporatedentityidentificationfrontend/views/capture_company_number_page.scala.html

```diff
@@ -56,15 +56,16 @@
             id = "companyNumber",
             name = "companyNumber",
             isPageHeading = false,
-            label = messages("capture-company-number.heading"),
-            hint = Some(Html(s"""<p class="govuk-body iei_hint">${messages("capture-company-number.hint")}</p>""")),
+            label = messages("capture-company-number.label"),
+            hint = Some(Html(s"""<p class="govuk-body govuk-hint">${messages("capture-company-number.hint")}</p>""")),
```

Comment:
It should work with just the messages("capture-company-number.hint") being used as the hint. This p tag looks like it was used as a way of overiding the default styling for hints (iei_hint class which we have removed)